Cargo.toml

```diff
@@ -33,5 +33,6 @@ chrono = "0.4.31"
 serde_yaml = "0.9.27"
 
 [features]
-all = ["log"]
+all = ["log", "fs"]
```

Comment:
I think we still need it in `all`

Reply:
> I think we still need it in `all`

Sorry, I didn't understand your point. Since our changes already include `fs` in `all`. 

Are you suggesting that we **should not include** `fs` in `all`?

Reply:
Strange, somehow I misunderstood that... sorry

Reply:
No worries :) 

I marked the same comment as `Resolved` in `rgb-interfaces`